### ISSUE TITLE
jackson does not consider @JsonSubTypes

### ISSUE THREAD

**Issue by jigarparekh80** [stoicflame/enunciate#335]
if parent class is marked with below annotation documentation for child classes  is not generated.
```
@JsonSubTypes({ @Type(value = Child1.class, name = "Child1"),
		@Type(value = Child2.class, name = "Child2") })
```
similar configuration is working for jaxb if class is marked with @XmlSeeAlso

**Comment by jigarparekh80** [comment]
workaround is adding @XmlRootElement to child classes.

**Comment by stoicflame** [comment]
Fixed at d25f9e4.

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
